src/services/organizationService.js

```diff
@@ -49,19 +49,27 @@ TODO:
 */
 
 function createToken(organization) {
+  //TODO: What payload will contain!
   const payload = {
     _id: organization.id,
     email: organization.email,
+    email: organization.email,
```

Comment:
Is the email duplicated?

Reply:
Yes, already fixed and published in the same branch.